src/email-verification/email-verification.controller.ts

```diff
@@ -3,17 +3,17 @@ import { EmailVerificationService } from './email-verification.service';
 import { ZaLaResponse } from 'src/common/helpers/response';
 import { SignupOTPDto } from './dto/email-token.dto';
 import { ApiOperation, ApiTags } from '@nestjs/swagger';
-@ApiTags("Email-verification")
+@ApiTags('Email-verification')
 @Controller('email-verification')
 export class EmailVerificationController {
   constructor(
     private readonly emailVerificatioService: EmailVerificationService,
   ) {}
 
   @Post('/confirm')
-  @ApiOperation({description: "Enter OTP send to your email"})
-  async verifyEmail(@Body() signupOTPDto: SignupOTPDto) {
-    const user = await this.emailVerificatioService.decodeEmailToken(signupOTPDto);
-    return ZaLaResponse.Ok<object>({user} , 'Profile created', 201);

```

Comment:
This is what we have now

![image](https://user-images.githubusercontent.com/84224449/192115049-97fd6e5a-978e-47d7-bf90-596471ca2c72.png)